lib/internal/fs.js

```diff
@@ -4,16 +4,17 @@ const Buffer = require('buffer').Buffer;
 const Writable = require('stream').Writable;
 const fs = require('fs');
 const util = require('util');
-const constants = process.binding('constants').fs;
-
-const O_APPEND = constants.O_APPEND | 0;
```

Comment:
I explored that a bit a couldn't find anything specific. I could be wrong, but I think the switch was more defensive just in case.